ksql-engine/src/test/resources/query-validation-tests/geodistance.json

```diff
@@ -0,0 +1,34 @@
+{
+  "tests": [
+    {
+      "name": "geo distance",
```

Comment:
this is great! can you also add another test for the case where radius isn't supplied at all?